### DISCUSSION TITLE
Not able to mirror torrent files.

### DISCUSSION THREAD

**Discussion by ayu-shah0912** [anasty17/mirror-leech-telegram-bot#1057] [Q&A]
When ever I try to mirror from torrent I get this error while the file is being uploaded, thanks in advace
{
 "error": {
  "errors": [
   {
    "domain": "global",
    "reason": "required",
    "message": "Login Required",
    "locationType": "header",
    "location": "Authorization"
   }
  ],
  "code": 401,
  "message": "Login Required"
 }
}

**Comment by anasty17** [comment]
Where this error came from??

**Reply by ayu-shah0912** [reply]
it came when I mirrored a torrent magnet link

**Reply by anasty17** [reply]
Yeah from bot logs?! If yes then send full logs!

**Reply by ayu-shah0912** [reply]
@ayu_shah HttpError 403 when requesting https://www.googleapis.com/drive/v3/files/1Sv0PNaODr41XG1Olbk3BdJcV8ASfPfRJ/permissions?supportsTeamDrives=true&alt=json returned "The user does not have sufficient permissions for this file.". Details: "[{'message': 'The user does not have sufficient permissions for this file.', 'domain': 'global', 'reason': 'insufficientFilePermissions'}]"

This is the exact message that I received from the bot

[https://www.googleapis.com/drive/v3/files/1Sv0PNaODr41XG1Olbk3BdJcV8ASfPfRJ/permissions?supportsTeamDrives=true&alt=json](url) this is the link to that log

**Reply by anasty17** [reply]
Read the error msg! It's clear. Use the upper part of ur body!

**Reply by ayu-shah0912** [reply]
i am noob sir, don't know how to solve this

**Reply by anasty17** [reply]
Read readme !

**Reply by ayu-shah0912** [reply]
don't know where to see and what to do

**Reply by anasty17** [reply]
Read all readme and follow all steps carefully

**Reply by ayu-shah0912** [reply]
I did everything just the way It was shown in the guide video in readme, still it doesn't work

**Reply by ayu-shah0912** [reply]
Edit* 
Files get mirrored, i am not getting the link message in the bot. Thanks again

**Comment by anasty17** [comment]
U know token.pickle! Or service accounts!!

**Reply by ayu-shah0912** [reply]
Flaged true on use service account and it works now, thank you sir

**Comment by ayu-shah0912** [comment]
So just has to make use service account to true, Thanks dev!

**Reply by anasty17** [reply]
If u r using service accounts then u should fill it true else leave it empty!
U DIDN'T READ!
